web-ui/src/components/menu/Menu.jsx

```diff
@@ -71,11 +71,11 @@ const directoryLinks = [
   ['/teams', 'Teams']
 ];
 
-const getFeedbackLinks = (isAdmin, isPDL, isSupervisor) => {
+const getFeedbackLinks = (isAdmin, isPDL, isSupervisor, hasReportPermission) => {
```

Comment:
So, in looking at this again...this isn't actually the right permission set. I'll identify it and drop it to you shortly.

Reply:
Okay, I'll work in the changes at that point.

Reply:
That PR is merged now.